bin-src/tasks/gitInfo.js

```diff
@@ -114,14 +114,19 @@ export const setGitInfo = async (ctx, task) => {
     try {
       const results = await Promise.all(baselineCommits.map((c) => getChangedFiles(c)));
       ctx.git.changedFiles = [...new Set(results.flat())];
-      ctx.log.debug(`Found changedFiles:\n${ctx.git.changedFiles.map((f) => `  ${f}`).join('\n')}`);
+      ctx.log.debug(
```

Comment:
I think we should change these logs to not be debug, at least in CI 🤷

Reply:
Done.